satzung.md

```diff
@@ -77,27 +77,35 @@
 * Eine Übertragung des Stimmrechts auf ein anderes Mitglied ist möglich, wenn die Stimmübertragung zumindest in Textform nachgewiesen wird. Ein Mitglied kann maximal das Stimmrecht von zwei weiteren Mitgliedern übertragen bekommen.
-* Die Mitgliederversammlung ist beschlussfähig, wenn sie ordnungsgemäß einberufen wurde und mindestens drei Siebtel der Stimmberechtigten vertreten sind. Sind weniger als drei Siebtel der stimmberechtigten Mitglieder vertreten, kann die Mitgliederversammlung erneut einberufen werden. Sie ist dann ohne Rücksicht auf die Zahl der vertretenen Mitglieder beschlussfähig. Auf diesen Umstand wird in der Einladung hingewiesen.
+* Die Mitgliederversammlung ist beschlussfähig, wenn sie ordnungsgemäß einberufen wurde.
 * Auf Antrag eines Mitgliedes ist die Abstimmung geheim durchzuführen.
 
 #### § 10 VORSTAND
 * Der Vorstand:
   * führt die Geschäfte des Vereins und fasst die erforderlichen Beschlüsse.
   * führt Aufzeichnungen über Ausgaben und Einnahmen des Vereins und stellt Spendenbescheinigungen aus.
 * Die Mitglieder des Vorstandes sind jeweils einzelvertretungsberechtigt.
-* Der Vorstand besteht aus entweder drei oder fünf Personen und wird auf ein Jahr gewählt. Eine Wiederwahl ist möglich.
+* Der Vorstand besteht aus entweder drei oder fünf Personen.
+* Der Vorstand wird auf ein Jahr gewählt. Eine Wiederwahl ist möglich.
+* Die Amtszeit des Vorstands beginnt mit der Wahl in der Mitgliederversammlung.
+* Nach Ende der Amtszeit bleibt der Vorstand bis zur nächsten Wahl kommissarisch im Amt.
 * Der Vorstand kann sich eine Geschäftsordnung geben, in der insbesondere Geschäftsverteilung innerhalb des Vorstandes und die gegenseitige Vertretung der Vorstandsmitglieder zu regeln ist.
+* Der Vorstand kann beschließen, Mitgliedern (inklusive Vorstands- und Beiratsmitgliedern) eine angemessene Aufwandsentschädigung für geleistete Tätigkeiten zu bezahlen, sofern dies die finanziellen Rahmenbedingungen des Vereins erlauben.  
 * Zu Sitzungen des Vorstandes ist eine Woche vorher in Textform zu laden. Mit dem Einverständnis aller Mitglieder des Vorstandes kann diese Frist verkürzt werden oder ganz entfallen.
 * Der Vorstand ist beschlussfähig, wenn die absolute Mehrheit der Vorstandsmitglieder anwesend ist.
 * Beschlüsse im Vorstand werden mit einfacher Mehrheit gefasst.
 * Beschlüsse des Vorstandes sind zu protokollieren.
-* Die Vorstandsmitglieder bleiben bis zu ihrer Amtsniederlegung oder Neuwahl im Amt.
+* Die Mitgliedschaft im Vorstand endet mit Ablauf der Amtszeit oder vorzeitig mit Beendigung der Vereinsmitgliedschaft oder durch Rücktritt aus dem Vorstand. Im Fall der vorzeitigen Beendigung bestimmen die verbleibenden Vorstandsmitglieder einen Nachfolger für die restliche Amtszeit.
```

Comment:
Motivation: Rücktritt eines Vorstandsmitglieds soll explizit geregelt werden. Der Fall ist bereits in der Vergangenheit aufgetreten und dabei war komplett unklar, wie wir damit umgehen sollen/müssen. Deswegen nun explizit geregelt.